src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -372,21 +382,25 @@ export class PanelComponent extends Disposable implements IThemable {
 				}`);
 			}
 			if (styles.titleActiveBorder && styles.titleActiveForeground) {
-				content.push(`.tabbedPanel.horizontal > .title .tabList .tab-header:focus,
-					.tabbedPanel.horizontal > .title .tabList .tab-header.active {
+				content.push(`.tabbedPanel.horizontal > .title .tabList .tab-header.active {
```

Comment:
Would suggest seeing if updating the `active` class and `titleActive*` names are going to be something easy to update as well